script.js

```diff
@@ -17,4 +17,122 @@ crossMenu2.addEventListener('click', closeMobMenu);
 
 navItem.forEach((menuItem) => {
   menuItem.addEventListener('click', closeMobMenu);
+});
+
+const openModal = document.querySelectorAll('[data-modal-target]');
+
+function openPopUp(popUpMod) {
+  popUpMod.classList.add('active');
+}
+
+openModal.forEach((button) => {
+  button.addEventListener('click', () => {
+    const popUpMod = document.querySelector(button.dataset.modalTarget);
+    openPopUp(popUpMod);
+  });
+});
+
+const modal1 = {
+  title: document.getElementById('title').textContent,
+  language1: document.getElementById('language1').textContent,
+  language2: document.getElementById('language2').textContent,
+  language3: document.getElementById('language3').textContent,
+  projectText: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+  live: 'https://www.google.com',
+  source: 'https://www.github.com',
+};
```

Comment:
- [ ] The pop-up should be responsive for both mobile and desktop screens.
![Screenshot from 2021-10-06 18-02-34](https://user-images.githubusercontent.com/65501535/136230304-376d710c-2aa1-4349-9019-ad1ae58575df.png)
![Screenshot from 2021-10-06 18-03-58](https://user-images.githubusercontent.com/65501535/136230543-249c8169-1c29-4ed9-a759-6e3af6fe0680.png)